test/built-ins/Temporal/PlainDateTime/limits.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime
+description: Numeric bounds check for constructor (string, plain object)
+features: [Temporal]
+---*/
+
+assert.throws(
+  RangeError,
+  () => new Temporal.PlainDateTime(-271821, 4, 19, 0, 0, 0, 0, 0, 0),
+  'negative year out of bounds'
+);
+assert.throws(
+  RangeError,
+  () => new Temporal.PlainDateTime(275760, 9, 14, 0, 0, 0, 0, 0, 0),
+  'positive year out of bounds'
+);
+
+const min = new Temporal.PlainDateTime(-271821, 4, 19, 0, 0, 0, 0, 0, 1);
+const max = new Temporal.PlainDateTime(275760, 9, 13, 23, 59, 59, 999, 999, 999);
```

Comment:
I'd use `TemporalHelpers.assertPlainDateTime` on these, as you did above in the PlainDateTime/from test.